lib/libutee/include/pta_system.h

```diff
@@ -160,4 +160,22 @@
  */
 #define PTA_SYSTEM_REMAP		9
 
+/*
+ * (libdl) dlopen(const char *filename, int flags)
```

Comment:
This has nothing to do with the interface of the PTA.

Reply:
I find it nice as it relates straight this PTA API with `dlopen()` std API.

Reply:
Perhaps as a note at the end of the comment then.

Reply:
OK I will move the line down